### DISCUSSION TITLE
UCoMP Fe XV may have been designed to the wrong wavelength

### DISCUSSION THREAD

**Discussion by bberkeyU** [NCAR/ucomp-pipeline#307] [General]
@detoma @jburkepile @mgalloy 
Per COSMO work, the FeXV wavelength in the atmosphere should be 705.959 (G Del Zanna) or 705.86 (1965 eclipse expedition), not the 706.2 in a vacuum we designed to quote in our poster.   

Our 706 pre-filter was designed to work at 706.3, with a slight tilt to center its bandpass on 706.2. To center the filter on 705.9nm, we would need an additional tilt.  The curve below looks at the wavelength shift vs filter tilt for 1074 and 706.
![image](https://github.com/user-attachments/assets/b1573983-4840-4f30-9c26-2c0810c57ac4)
This is based on a glass index of 1.45 and the following equation:
![image](https://github.com/user-attachments/assets/65ed3caf-1d53-4e23-95d6-58f36127e5ef)

I am unsure if we measured the filter tilts, but assuming they all have similar mechanical mounting, I expect all the filters to have a similar tilt.  The 1074 filter design required a 0.2nm shift to bring 1074.7 to the center of the filter, which suggests the filters were designed for a ~1.5-degree tilt.  This should shift the 706 filter to about ~.18nm.  To achieve the full 0.4nm from filter design to 705.9nm, we may need to further shift the filter to 2.5 degrees, which is likely within the starlight express filter wheel. 

However, we should still be ok with the existing tilt 0.18nm shift; this puts the 705.9 +/- .05 nm tunings on the shoulder of the filter but still close enough to the top that we should reasonably onband emission and good red continuum. 
![image](https://github.com/user-attachments/assets/5dd418c3-b8ad-4cb0-95d1-829230420b04)

During the commission phase, we didn't do any scans of 705.9. This was wider than ever, looking at our 21 or 13-point coronal scans.  However, considering the on-band continuum spacing of 0.47nm, we scanned the continuum over this region when we tuned to 706.37 in a few of these wider scans.

Below, I have plotted the Background I (continuum) channels from some 706 data taken in Feb 2022.  Unfortunately, we were messing with the wavelength offset during this period, so the offsets jumped from 1.68 on Feb 11th to 1.86 on Feb 13th, so the channel with the structure moved about 0.2nm during this period as well. 

Data use: 20220215.202145.ucomp.706.l1.p11.fts  and 20220211.192844.ucomp.706.l1.p5.fts
![image](https://github.com/user-attachments/assets/1f7d52e2-7962-4dac-ac44-7be00b360cf8)
![image](https://github.com/user-attachments/assets/0350e476-902c-444c-8dc5-b4681aa14a6e)

Are these bright structures the FeXV data? To me, they look more like chromospheric than coronal (Blobs vs. loops). If this is the correct information, it might be interesting to try doing a special pipeline run for our wide scans that swaps the logic of how it handles the on- and off-band channels.  

This is one of many examples of this 705.9(ish) emission data in UCoMP. We have just assumed it was contamination from a chromospheric source, but we have yet to be able to figure out which one.     The next steps for me are to try and fit the intensity variation seen in our flats and the solar/telluric lines seen in 706, so hopefully, we can nail down the exact wavelength for this emission.

**Comment by detoma** [comment]
According to NIST, the Ritz vacuum wavelength of Fe XV is 706.2, the observed wavelength in vacuum is 706.05nm
According to NIST, the Ritz air wavelength of Fe XV is 706.0nm, the observed wavelength in air (Eclipse of 1965) is 705.86nm
According to CHIANTI the vacuum wavelength of Fe XV is 706.215nm, which correspond to 706.02nm in air, based on the correction of Birch and Down, Metrology 1994. Note that it differs slightly from the NIST correction. Difference = 0.05nm. There may be better correction to go from vacuum to air. I used two different codes and got me the same result.
Giulio Del Zanna reports a shorter wavelength in air for Fe XV of 705.959nm based on Wlérick & Fehrenbach (1963).

The table below is from the CHIANTI database. Wavelengths are in vacuum. The last two columns are T and Intensity
<img width="851" alt="Screenshot 2024-11-21 at 3 52 00 PM" src="https://github.com/user-attachments/assets/b53a57c0-72dd-4056-8d5d-ce1b92276c08">

In summary, we do not know the precise wavelength of Fe XV. Values **in air** based on theory and measurements vary between 705.86nm and 706.02nm. The 706.20nm seems either mix-up between vacuum and air or a typo of the CHIANTI air value.

The HeI wavelengths reported by NIST are the same as in CHIANTI, i.e. 706.71nm, 706.72nm, and 706.77nm. in vacuum. They correspond in air to 706.52nm, 706.52nm, and 706.57nm.	

The images Ben uploaded looked like the images of prominence that were on the Sun on those days. I agree that it looks like chromospheric and not coronal emission and are likely coming from HeI.

**Reply by bberkeyU** [reply]
I don't think we know the tuning/filter bandpass to better that 2 or 3 nm.  But I don't think we should be seeing in leakage from our side the prefilter. I think we see the helium in 1079 because it is so broad but this emission is relatively narrow we can see it go thru it's peak in these scans.  

The prefilters also have really good rejections out side the bandpass. if the lyot filter is getting constructive interference at some other wavelength with these tuning voltages the source would need to be really really bright.

**Reply by bberkeyU** [reply]
What is the best atlas of bright chromospheric lines? 

Here is the prefilter data, and the transmission goes close to zero outside the narrow window around 706.
Wideband pre-filter 
![image](https://github.com/user-attachments/assets/dac03712-d12f-4b7b-8b7a-1630d7ba0df4)
Narrowband prefilter
![image](https://github.com/user-attachments/assets/73e00cb1-5186-4e49-ae3a-1a88adf6488d)

The emission also looks even more Chromospheric in the 20240330 706 data.  Here I did a one beam sudo I image.
RCAM data were dark subtracted, and I averaged across the 4 modulations.  And then, the 706.36 ONBAND data was subtracted from the OFFBAND, giving a resultant image of the 705.9.   In it, we see the bright near-saturated cores of the active regions and that annoying blooming under the occulter that made the 656 and 1083 look funky.
![image](https://github.com/user-attachments/assets/dc03252d-042d-4e15-8b7b-7eb2b880a013)

**Reply by bberkeyU** [reply]
According to NIST https://physics.nist.gov/PhysRefData/Handbook/Tables/heliumtable2.htm there is a chrompsheric HeI emission line at 706.51771 nm in air.  And if I am reading the tables correctly this FeXV has a comparable (100) relative brightness to h-alpha (180) or the weakest of the HeI 1083 lines (160,600,1000).  I am not sure how we missed it. I assumed this plot would have included HeI lines. ![image](https://github.com/user-attachments/assets/9bd68b6b-d3d3-4447-b2e5-23d6ce6fd7f9)

Assuming that the emission we see in the above image is 706.51, I will try to use this to fit the wave-off value, check out how we're doing with our 705.9 tuning, and compare that with our previous assessments of the wave-off.

**Comment by bberkeyU** [comment]
The source is relatively in focus so it probably isn't far from 706nm.

**Comment by detoma** [comment]
As discussed yesterday at the UCoMP meeting, we do not likely know the wavelength calibration in the 706nm channel. We plan to take wide scans to search again for the Fe XV line ( and the HeI line) and also to take wide scans in the flats to get a solar spectrum for wavelength calibration/verification.

I will update the lines nominal wavelength above for the Fe XV and HeI using CHIANTI and NIST.

What is confusing is that HeI should be on the red side of the Fe XV line but in early February there is some chromospheric signal in our data on the blue of the line and at the "nominal" 706.2nm. This is probably a problem with the wavelength scale that we can resolve by taking broad scans when we reopen.  It is seems for earlier data in 2021 we also have the wrong wavelength scale, i.e. lines are not where expected.

Later in the mission, it is hard to tell because we only have 3 points.  We are probably close to be centered on the Fe XV line  but we have the wrong wavelength for it. The chromospheric signal is in the continuum, or ~0.4-0.5nm from the line, as expected. 

Based on the AIA images in Fe XIV at 211nm and Fe XVI at 335nm, we do not expect prominences to show in emission in Fe XV. He I should be a pretty string line (but interestingly I could not find any publication on prominences in HeI 706.7nm, the only papers that mentioned this line were for other stars). There is a much fainter Al II line blue of FeXV which is at the same temperature of HeI.
